lib/propolis/src/hw/nvme/mod.rs

```diff
@@ -496,7 +511,11 @@ pub struct PciNvme {
 
 impl PciNvme {
     /// Create a new pci-nvme device with the given values
-    pub fn create(serial_number: String, log: slog::Logger) -> Arc<Self> {
+    pub fn create(
+        serial_number: String,
+        mdts: Option<u8>,
```

Comment:
I know we eventually want this coordinated with the block layer but given both call sites just hardcode the same value now I'm inclined to move it into the body here. We already have a whole bunch of other controller config/capabilities hardcoded there.

Reply:
I'd like to leave it as-is for now.  It's good incentive to make sure a tunable like this is addressed when some of the other migration/block changes go in.

Reply:
sounds good